client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -33,6 +37,8 @@ const PlansColumn = ( { duration, onPlanClick, productType, siteId }: PlanColumn
 	const currentPlan =
 		useSelector( ( state ) => getSitePlan( state, siteId ) )?.product_slug || null;
 
+	const optionsFromCurrentPlan = getAllOptionsFromSlug( currentPlan ) || [];
```

Comment:
If a user owns `jetpack_backup_daily_monthly`, we don't want to display the Jetpack Backup option card, therefore, we search for all slugs of the Jetpack Backup option cards (there are two, a monthly and a yearly).

Reply:
There's a small TS error at this line.

Reply:
Good catch!